queries/javascript.core.scm

```diff
@@ -681,6 +681,13 @@
   (#not-parent-type? @statement export_statement)
 )
 
+(program) @statement.iteration
+
+(statement_block
+  "{" @statement.iteration.start.endOf
+  "}" @statement.iteration.end.startOf
+)
+
```

Comment:
@AndreasArvidsson we should prob factor this out into its own PR with a couple tests, no?

Reply:
done
https://github.com/cursorless-dev/cursorless/pull/2144